src/riak_pb_ts_codec.erl

```diff
@@ -124,6 +124,8 @@ encode_cell(V) when is_boolean(V) ->
 encode_cell({time, V}) ->
     #tscell{timestamp_value = V};
 encode_cell(undefined) ->
+    #tscell{};
+encode_cell([]) -> %% NULL Cell
```

Comment:
Retrieval. We put the `[]` in here: https://github.com/basho/riak_pb/pull/152/files#diff-5a85f13716f1297dfbd5fc46fd4c48ecR176